src/components/editor/metaform-editor-right-drawer-feature.tsx

```diff
@@ -517,14 +523,12 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
   /**
    * Render member group role options
    *
-   * @param field selected metaform field
    */
-  const renderMemberGroupPermissionSelect = (field: MetaformField) => {
-    if (memberGroupOptIndex === undefined || selectedMemberGroupId === undefined) {
+  const renderMemberGroupPermissionSelect = () => {
+    if (memberGroupOptIndex === undefined || selectedMemberGroupId === undefined || selectedMemberGroupId === NOT_SELECTED || !selectedField) {
       return null;
     }
```

Comment:
Why using the selected field instead of the field from function prop? the field is just the selected field after null check

Reply:
Fixed that back